exchange/bidder.go

```diff
@@ -87,6 +87,8 @@ type pbsOrtbSeatBid struct {
 	// if len(bids) > 0, this will become response.seatbid[i].ext.{bidder} on the final OpenRTB response.
 	// if len(bids) == 0, this will be ignored because the OpenRTB spec doesn't allow a SeatBid with 0 Bids.
 	ext json.RawMessage
+
+	warnings map[openrtb_ext.BidderName][]openrtb_ext.ExtBidderMessage
```

Comment:
The `pbsOrtbSeatBid` object is already bidder specific. The mapping from bidder to a map a bidder errors is make in exchange/exchange.go#makeExtBidResponse.

```
if len(responseExtra.Errors) > 0 {
	bidResponseExt.Errors[bidderName] = responseExtra.Errors
}
if len(errList) > 0 {
	bidResponseExt.Errors[openrtb_ext.PrebidExtKey] = errsToBidderErrors(errList)
}
```

I was originally thinking this would look something like:
```
if errs := errortypes.FatalOnly(responseExtra.Errors); len(errs) > 0 {
	bidResponseExt.Errors[openrtb_ext.PrebidExtKey] = errsToBidderErrors(errs)
}

if warnings := errortypes.WarningOnly(responseExtra.Errors); len(warnings) > 0 {
	bidResponseExt.Warnings[openrtb_ext.PrebidExtKey] = errsToBidderWarnigs(warnings)
}
```

Reply:
Good point about bidder specific warning. I modified it to list.
To use one list for both Errors and Warning I need to use `error` (or consolidated struct) instead of  `openrtb_ext.ExtBidderMessage`. Should I process with this?